src/datalad_installer.py

```diff
@@ -344,7 +344,7 @@ def parse_args(
         except GetoptError as e:
             raise UsageError(str(e), self.component)
         kwargs: Dict[str, Any] = {}
-        for (o, a) in optlist:
+        for o, a in optlist:
```

Comment:
I need to learn Python - why any of the two changes matter for tests?

Reply:
This was caused by updating black in pre-commit (which I did because of the isort installation bug).